updating/updating-restricted-network-cluster.adoc

```diff
@@ -123,7 +135,27 @@ include::modules/update-service-install-cli.adoc[leveloffset=+2]
 
 include::modules/update-service-graph-data.adoc[leveloffset=+2]
 
-include::modules/update-service-mirror-release.adoc[leveloffset=+2]
+[id="update-service-mirror-release_{context}"]
+=== Mirroring the {product-title} image repository
+
+Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+//The module below is being used twice in this assembly, so this instance needs to have a unique context set in order for its ID to be unique. In the future, if and when the two main sections of this webpage are split into their own assemblies/pages, the context attributes below can be removed.
+
+:!context:
+:context: OSUS-restricted-network-cluster
```

Comment:
If you do end up keeping this setting/unsetting of context, then this OSUS should be lowercase.